### DISCUSSION TITLE
Connecting to the AI Deck

### DISCUSSION THREAD

**Discussion by rithwikjayanth233** [bitcraze/discussions#766] [Q&A]
Hi,

I have crazyflie drone and am trying to connect it to an AI Deck (https://www.bitcraze.io/products/ai-deck/). I followed the steps given in https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/, but I didn't get the ' ESP32: I (910) SYS: Initialized' message in the console tab. Instead, I get 'ESP32: I (505) SYS: Initialized'. I am not sure if this is right. Any help would be great!

**Comment by krichardsson** [comment]
Hi, 

the log you see originates from the [system_init() function](https://github.com/bitcraze/aideck-esp-firmware/blob/main/main/system.c#L81) in the ESP. That is, it uses the default logging system in the ESP.

The log information is passed on to the Crazyflie using CPX and is finally [printed in the console log by the cpx module](https://github.com/bitcraze/crazyflie-firmware/blob/master/src/modules/src/cpx/cpx.c#L123) 

To decode the full message `ESP32: I (505) SYS: Initialized`
1. `ESP32` is added to indicate the source
1. `I` = information (the `ESP_LOGI()` was used)
1. `(505)` a time stamp (ms since boot I think)
1. `SYS` the log module in the ESP
1. `initialize` the actual log message

As you can see the difference you get is the time stamp, probably nothing to worry about.
